.travis.yml

```diff
@@ -38,6 +38,15 @@ matrix:
     - env: CABALVER=1.24 GHCVER=8.0.1
       compiler: ": #GHC 8.0.1"
       addons: {apt: {packages: [cabal-install-1.24,ghc-8.0.1], sources: [hvr-ghc]}}
+    - env: CABALVER=1.24 GHCVER=8.2.2
+      compiler: ": #GHC 8.2.2"
+      addons: {apt: {packages: [cabal-install-1.24,ghc-8.2.2], sources: [hvr-ghc]}}
+    - env: CABALVER=1.24 GHCVER=8.4.3
+      compiler: ": #GHC 8.4.3"
+      addons: {apt: {packages: [cabal-install-1.24,ghc-8.4.3], sources: [hvr-ghc]}}
+    - env: CABALVER=1.24 GHCVER=8.6.1
+      compiler: ": #GHC 8.6.1"
+      addons: {apt: {packages: [cabal-install-1.24,ghc-8.6.1], sources: [hvr-ghc]}}
```

Comment:
please use `cabal-install-2.4` for GHC-8.6.1, 2.2 for 8.4, 2.0 for 8.2. Otherwise things may fail.

Reply:
I have updated the cabal versions to match the newer ghc versions.